src/component/utils/getElementForBlockType.js

```diff
@@ -17,9 +17,15 @@ import type {DraftBlockType} from 'DraftBlockType';
 function getElementForBlockType(blockType: DraftBlockType): string {
   switch (blockType) {
     case 'header-one':
-      return 'h2';
```

Comment:
I believe this was a typo ? so I updated the header-one to return h1. Is that a problem or was that the intendend behaviour ? If so I can amend the PR ?